test/e2e/framework/psp_util.go

```diff
@@ -97,7 +97,7 @@ var (
 )
 
 func CreatePrivilegedPSPBinding(f *Framework, namespace string) {
-	if !IsPodSecurityPolicyEnabled(f) {
+	if !IsPodSecurityPolicyEnabled(f) || !IsRBACEnabled(f) {
```

Comment:
If the PSP is enabled, but RBAC is disabled, it probably means the cluster is using ABAC or allow all. In either case, we still want to create the PSP to avoid breaking things, we just don't need to bind it, right?

Reply:
In our case we are using a custom authorization webhook instead of RBAC which is ofc. a bit special. I will try and see how it works if only the binding is skipped.

Reply:
@tallclair changed it to only create the binding if RBAC is enabled, but always create the PSP PTAL.